components/o-topper/src/scss/themes/_deep-landscape.scss

```
@@ -43,37 +50,32 @@
 		}
 	}
 
-	
+
 	@include oGridRespondTo( $until:S) {
 		.o-topper__content {
 			width:100%;
-			align-self: end;
-			grid-column: 1 / span 2;
-			padding:20px;
+		  	margin: 0;
+		  	padding: 20px;
 		}
 	}
-	
+
 	@include oGridRespondTo($from:S , $until:M) {
 		.o-topper__content {
 			width:75%;
-			grid-column: 1/span 2;
-			margin: auto auto auto 10%;
 		}
 	}
 	@include oGridRespondTo($from:M , $until:L) {
 		.o-topper__content {
 			width:50%;
-			grid-column: 1/span 2;
-			margin: auto auto auto 10%;
 		}
 	}
 
 	@include oGridRespondTo(L) {
 		grid-template-rows: 66vw 1fr;
 
 		.o-topper__content {
-			grid-column: 1;
-			margin: auto;
+		  	grid-column: 1;
+		  	margin: 0 0 72px 0;
```

```suggestion
		  	margin: 0 0 72px;
```

The linter is erroring here because the last `0` is superfluous:
https://stylelint.io/user-guide/rules/shorthand-property-no-redundant-values/

I actually disagree with this rule, and think we should look to remove it, because it doesn't really matter and I believe what you wrote is more clear.